.devcontainer/post_create.sh

```diff
@@ -14,14 +14,18 @@ echo 'export PATH="$HOME/.local/bin:$PATH"' >> "$bashrc_path"
 cat $HOME/.zshrc
 export PATH="$HOME/.local/bin:$PATH"
 
-# Install Trivy
-export TRIVY_VERSION=0.63.0
-wget https://github.com/aquasecurity/trivy/releases/download/v${TRIVY_VERSION}/trivy_${TRIVY_VERSION}_Linux-64bit.deb
-sudo dpkg -i trivy_${TRIVY_VERSION}_Linux-64bit.deb
-rm trivy_${TRIVY_VERSION}_Linux-64bit.deb
-
 # Install Trivy plugins
 trivy plugin install mcp
 
 # Install CVE Search
-cd mcp_servers/cve-search_mcp && uv sync
\ No newline at end of file
+pushd mcp_servers/cve-search_mcp || exit
```

Comment:
if we are doing the pushd approach we need to handle errors with uv sync or `uv sync || exit`